src/Http/Error.php

```diff
@@ -45,13 +45,30 @@ public function __construct(
         string $message = ''
     ) {
 
+        $factory = \NDB_Factory::singleton();
```

Comment:
No other response type is using globals, you can just pass the parameters you want in as function arguments.